sound/hybrid_font.h

```diff
@@ -548,7 +548,8 @@ class HybridFont : public SaberBase {
     return current_effect_length_;
   }
 
-
+  void SB_LowBatt() override { PlayCommon(&SFX_lowbatt); }
```

Comment:
We still need the talkie fallback if SFX_lowbatt is not available, guess I should have been more clear.

Reply:
ok i have added that